libgobuster/http.go

```diff
@@ -135,6 +137,10 @@ func (client *HTTPClient) makeRequest(fullURL, host string, data io.Reader) (*ht
 		req.Header.Set("Cookie", client.cookies)
 	}
 
+	if client.host != "" {
```

Comment:
there is already a host parameter passed in. This should be used instead of introducing new struct fields

Reply:
Should be good now